build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
 }
 
 group = 'com.synopsys.integration'
-version = '8.11.0-SNAPSHOT'
+version = '8.11.0-SIGQA6-SNAPSHOT'
```

Comment:
8.11.0-SIGQA2-SNAPSHOT is the current version in the feature branch (for resolving your conflict). I'll bump it up to 9.x after this PR is merged - thanks.